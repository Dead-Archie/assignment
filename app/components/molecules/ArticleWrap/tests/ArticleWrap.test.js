import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import { ArticleWrap } from '../ArticleWrap';

const sampleProps = {
  title: 'test',
  children: [],
  isLeft: true,
  subtitle: 'test',
};
const setUp = props => {
  return shallow(<ArticleWrap {...props} />);
};

describe('<ArticleWrap />', () => {
  let ArticleWrapComponent;
  beforeEach(() => {
    ArticleWrapComponent = setUp(sampleProps);
  });

  test('should render correctly', () => {
    expect(ArticleWrapComponent).toMatchSnapshot();
  });
});
