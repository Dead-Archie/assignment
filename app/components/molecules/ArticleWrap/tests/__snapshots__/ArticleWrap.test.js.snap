// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArticleWrap /> should render correctly 1`] = `
<article
  className="text-left"
>
  <div
    className="inner"
  >
    <h2>
      test
    </h2>
    <div
      className="sub-heading"
    >
      test
    </div>
    <div
      className="container-wrap m-t-30"
    />
  </div>
</article>
`;
